kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportPage.vue

```diff
@@ -119,72 +119,51 @@
       ReportsLearnerHeader,
     },
     mixins: [commonCoach, commonCoreStrings],
-    setup() {
-      const { saveTabsClick, wereTabsClickedRecently } = useCoachTabs();
-      return {
-        saveTabsClick,
-        wereTabsClickedRecently,
-      };
-    },
     data() {
       return {
-        REPORTS_LEARNERS_TABS_ID,
-        ReportsLearnersTabs,
+        limit: 10,
```

Comment:
Maybe naming this to be more explicitly would be great for clarity, as done with 'quizLimit'?